x-pack/plugins/security_solution/public/common/components/import_data_modal/index.tsx

```diff
@@ -81,21 +103,46 @@ export const ImportDataModalComponent = ({
         const importResponse = await importData({
           fileToImport: selectedFiles[0],
           overwrite,
+          overwriteExceptions,
           signal: abortCtrl.signal,
         });
 
-        if (importResponse.success) {
-          addSuccess(successMessage(importResponse.success_count));
-        }
-        if (importResponse.errors.length > 0) {
-          const formattedErrors = importResponse.errors.map((e) => failedDetailed(e.error.message));
-          const error: Error & { raw_network_error?: object } = new Error(
-            formattedErrors.join('. ')
-          );
-          error.stack = undefined;
-          error.name = 'Network errors';
-          error.raw_network_error = importResponse;
-          addError(error, { title: errorMessage(importResponse.errors.length) });
+        // if import includes exceptions
+        if (showExceptionsCheckBox) {
+          // rules response actions
+          if (importResponse.success) {
```

Comment:
looks like lines 113-119 and 139-145 are the same. so maybe we can move it to 110 and remove the else block